src/videotile/VideoTileState.ts

```diff
@@ -5,25 +5,108 @@
  * [[VideoTileState]] encapsulates the state of a [[VideoTile]]
  */
 export default class VideoTileState {
+  /**
+   * The unique identifier for the [[VideoTile]] managed by [[VideoTileController]]. Each attendee can have at most one tileId.
+   */
   tileId: number | null = null;
+
+  /**
+   * Indication of whether tile is associated with local video.
+   */
   localTile: boolean = false;
+
+  /**
+   * Indication of whether local tile has started to play, if this [[VideoTileState]] is to be associated with local tile.
```

Comment:
Nit: Should we remove "if this [[VideoTileState]] is to be associated with local tile" and keep it to-
 "Indication of whether the tile associated with the local attendee has started to play"

Reply:
will do, thanks